shell/platform/fuchsia/flutter/tests/integration/README.md

```diff
@@ -0,0 +1,84 @@
+# `flutter integration tests`
+
+## Configure and build fuchsia
+
+```shell
+$ cd "$FUCHSIA_DIR"
+$ fx set terminal.x64
+$ fx build
+```
+
+## Build the test
```

Comment:
I might be wrong but it feels like the build and run test parts could be encapsulated in a build_and_run_test.sh script that takes the name of the integration test folder to run. There might be a switch statement that chooses what package names to publish based on the name of the integration test folder, but besides that it feels the same from test to test.

It feels like it would be a more convenient workflow to do something like:

```
$ENGINE_DIR/tools/fuchsia/devshell/build_and_run_test.sh flutter-embedder-test
```

And have it build, publish everything we need and then run the test.

Reply:
Will write this and update instructions

Reply:
FWIW I did not get a chance to write this @akbiggs Can we file a bug and push a follow-up CL for it?

Reply:
No worries! fxb/107344